src/test/java/com/kodilla/ecommercee/domain/UserEntityTest.java

```diff
@@ -0,0 +1,123 @@
+package com.kodilla.ecommercee.domain;
+
+import com.kodilla.ecommercee.repository.CartRepository;
+import com.kodilla.ecommercee.repository.OrderRepository;
+import com.kodilla.ecommercee.repository.UserRepository;
+import org.jetbrains.annotations.NotNull;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@SpringBootTest
+public class UserEntityTest {
+
+    @Autowired
+    private UserRepository userRepository;
+    @Autowired
+    private CartRepository cartRepository;
+    @Autowired
+    private OrderRepository orderRepository;
+
+    private User prepareUserData() {
+        Order order1 = new Order();
+        Order order2 = new Order();
+        Cart cart = new Cart();
+        User user = new User();
+
+        cartRepository.save(cart);
+        orderRepository.save(order1);
+        orderRepository.save(order2);
+
+        order1.setDateOfOrder(LocalDate.now());
+        order1.setOrderStatus("done");
+        order1.setUser(user);
+        order1.setOrdersItems(new ArrayList<>());
+
+        order2.setDateOfOrder(LocalDate.now());
+        order1.setOrderStatus("done");
+        order2.setUser(user);
+        order2.setOrdersItems(new ArrayList<>());
```

Comment:
Are You sure that `order1.setOrdersItems(new ArrayList<>());` and `order2.setOrdersItems(new ArrayList<>());` are necessary? I think that the `ordersItems `field is already initialized with an empty list in the Order entity's constructor.

Reply:
Without tests were crashing :/